micro-airtable-api.test.js

```diff
@@ -0,0 +1,77 @@
+const http = require('http');
+const got = require('got');
+const listen = require('test-listen');
+const handler = require('./index');
+
+const describeIfEnv = (process.env.AIRTABLE_API_KEY && process.env.AIRTABLE_BASE_ID) ? describe : xdescribe;
+const baseConfig = {
+  airtableBaseId: process.env.AIRTABLE_BASE_ID,
+  airtableApiKey: process.env.AIRTABLE_API_KEY,
+};
+
+const getExistingPosts = async (client) => {
+  return (await client.get('/v0/Posts', {json: true})).body;
+};
+
+describeIfEnv('micro-airtable-api', () => {
+
+  describe('when `allowedMethods="*"`', () => {
+    const config = {...baseConfig, allowedMethods: '*'};
+    let server;
+    let client;
+
+    beforeAll(async () => {
+      server = http.createServer(handler(config));
+      const baseUrl = await listen(server);
+      client = got.extend({ baseUrl });
+    });
+
+    describe('GET /v0/Posts', () => {
+      it('returns successful response', async () => {
+        await client.get('/v0/Posts');
+      });
+    });
+
+    describe('POST /v0/Posts', () => {
+      it('returns successful response', async () => {
+        await client.post('/v0/Posts', {
+          json: true,
+          headers: {'Content-type': 'application/json'},
```

Comment:
If you set `json: true`, then you don't need to set this header manually.

From the [got docs](https://github.com/sindresorhus/got#json).

> If set to true and Content-Type header is not set, it will be set to application/json.